examples/ng/doc/lifecycle-hooks/pubspec.yaml

```diff
@@ -3,13 +3,13 @@ description: Lifecycle Hooks
 version: 0.0.1
 
 environment:
-  sdk: '>=2.5.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
```

Comment:
Do you have a 2.12 SDK to check whether Angular 7 can actually run for Dart versions below 2.14? I think theoretically Angular 7 is only for 2.14+ (see [here](https://github.com/angulardart/angular/blob/e943f6f0b8f8ae0d64c2c7935d4554f41d86e2cd/angular/pubspec.yaml#L8)), but I never verified it by myself.